libweb3jsonrpc/Debug.cpp

```diff
@@ -163,10 +163,10 @@ Json::Value Debug::debug_accountRange(
 
         Json::Value addressList(Json::objectValue);
         for (auto const& record : addressMap.first)
-            addressList[toString(record.first)] = toString(record.second);
+            addressList[toString(record.first)] = record.second.hex();
```

Comment:
I didn't try, but I don't think this adds a prefix.

Reply:
Note that key should be prefixed, too.

Reply:
Ah, I should use `toHexPrefixed `instead of `toHex`